src/main/resources/templates/start.html

```diff
@@ -1,19 +0,0 @@
-<!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org">
-<head>
-<link rel="stylesheet" href="/CSS/start.css">
-<style>
-body{
-	background: black;
-	color: white;
-}
-</style>
-<meta charset="ISO-8859-1">
-<title>PlantPlaces.com</title>
-</head>
-<body>
-<h1>Welcome to plant places</h1>
-<br/>
-<p th:text="${specimenDTO}"/>
-</body>
-</html>
```

Comment:
I went ahead and removed this start.html file. It looks like you guys already had a good homepage with the platforms html page, and the start.html file contained references to the plant places project, which I don't think you guys meant to leave in.